### ISSUE TITLE
Doesn't show the Lottie

### ISSUE THREAD

**Issue by rokib-rajon** [chadly/gatsby-remark-lottie#2]
I have installed the plugin following your instruction. But it cannot show the Lottie, only the alt text is visible. 
This is how I am trying implement on my markdown file.
`
---
title: Home
sections:
  - type: hero_section
    title: Welcome to my Portfolio
    subtitle: Optional hero section subtitle
    content: >-
      A designers world, see it and be sure about skills I have.
    image: designer.json
    image_alt: Hero section placeholder image
    media_position: right
    media_width: fifty
---
 `

**Comment by Abhiram716** [comment]
@rokib-rajon did you find any solution for the above problem? please share if you did
